sdk/src/loader_instruction.rs

```diff
@@ -8,23 +8,22 @@ use crate::{
 pub enum LoaderInstruction {
     /// Write program data into an account
     ///
-    /// * key[0] - the account to write into.
-    ///
-    /// The transaction must be signed by key[0]
+    /// # Account accesses
+    ///   * 0. [WRITE, SIGNER] Account to write to
     Write {
         offset: u32,
```

Comment:
Should we also add comments for these members?

Reply:
Done